src/reuse/modules/nonUi5/element.ts

```diff
@@ -204,7 +207,8 @@ export class ElementModule {
         return await $(selector);
       }
     } catch (error) {
-      throw new Error(`Function 'getById' failed. Element with id '${id}' not found. ${error}`);
+      return this.ErrorHandler.logException(error,`Element with id '${id}' not found`)
+      //throw new Error(`Function 'getById' failed. Element with id '${id}' not found. ${error}`);
```

Comment:
same here

Reply:
The changes reverted back and will add this changes in next PR